homepage/homepage.css

```diff
@@ -0,0 +1,252 @@
+@import url(../nav1.css);
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.gallery{
+    margin: 10px auto;
+    display:grid;
+    grid-template-columns: repeat(3,1fr);
+    column-gap:5px;
+    row-gap: 5px;
+   
+
+}
+main h2{
+    text-align: center;
+    margin: 30px;
+    font: Futura, sans-serif;
+    line-height:24.5px;
+    color:red;
+
+
+}
+.gallery_img{
+    width:200px;
+    height:250px;
+    
+}
+.gallery_img:hover{
+    transform: scale(1.04);
+}
+.gallery_jersey{
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+    position: relative;
+    justify-content: center;
+    background-color: white;
+    padding: 10px;
+
+}
+.gallery_jersey>p{
+    text-align: center;
+    font-size: 1.2rem;
+    font-weight: bold;
+    color:white;
+    
+}
+
+.btn_jersey{
+    background-color: white;
+    color: green;
+    padding: 3px 8px;
+    text-align: center;
+    display: inline-block;
+    font-size: 1rem;
+    margin: 10px 20px;
+    cursor: pointer;
+    text-decoration: none;
+
+}
+
+.jersey_text_ind{
+    background-color:orangered
+}
+.jersey_text_eng{
+    background-color:#61ACC3
+}
+.jersey_text_sl{
+    background-color:#048CE8
+}
+.jersey_text_nz{
+    background-color:#121212
+}
+.jersey_text_aus{
+    background-color:#F0EF21
+}
+.jersey_text_sa{
+    background-color:#77D696
+}
+
+/*Must Have Section*/
+section{
+    margin: 0 auto;
+}
+.must-have{
+    max-width: 1200px;
+    margin: 2rem auto;
+    height: 40px;
+    display: flex;
+    height:400px;
+    gap: 15px;
+    box-shadow: rgba(240, 46, 170, 0.4) 0px 5px, rgba(240, 46, 170, 0.3) 0px 10px, rgba(240, 46, 170, 0.2) 0px 15px, rgba(240, 46, 170, 0.1) 0px 20px, rgba(240, 46, 170, 0.05) 0px 25px;
+    justify-content: center;
+}
+section h2{
```

Comment:
Give a classname to h2 and give the styling to itself and try not to give styling on the element tag